rcl/include/rcl/macros.h

```diff
@@ -20,14 +20,10 @@ extern "C"
 {
 #endif
 
-#ifndef _WIN32
 /// Ignored return values of functions with this macro will emit a warning.
-# define RCL_WARN_UNUSED __attribute__((warn_unused_result))
-#else
-# define RCL_WARN_UNUSED _Check_return_
-#endif
+#define RCL_WARN_UNUSED RCUTILS_WARN_UNUSED
 
-#define RCL_UNUSED(x) (void)(x)
+#define RCL_UNUSED(x) RCUTILS_UNUSED(x)
```

Comment:
@dirk-thomas this change seems unrelated. No need to split in a separate PR, a different commit + rebase-merge would do too.

Reply:
Reverted in a1c8a5b1d0c80946805b81e5016cf7977172bf63.